src/helpers/calculateOnStock.js

```diff
@@ -0,0 +1,9 @@
+import calculateProductsSold from "./calculateProductsSold.js";
+import calculateOriginalStock from "./calculateOriginalStock.js";
+
+function calculateOnStock(inventoryList) {
+    const totalSold = (calculateProductsSold(inventoryList));
```

Comment:
De buitenste ronde haken zijn in principe niet nodig
```suggestion
    const totalSold = calculateProductsSold(inventoryList);
```